src/main/java/exception/NarutoException.java

```diff
@@ -0,0 +1,128 @@
+package exception;
+
+/**
+ * NarutoException is an abstract class that extends the Exception class.
+ * It represents a custom exception in the Naruto application.
+ */
+public abstract class NarutoException extends Exception {
+
+    /**
+     * Enum representing the different types of errors that can occur in the Naruto application.
+     */
+    public enum ErrorType {
+        EMPTY_COMMAND,
+        EMPTY_DEADLINE,
+        EMPTY_EVENT,
+        INVALID_DEADLINE,
+        INVALID_EVENT,
+        INVALID_COMMAND,
+        INVALID_INDEX,
+        FILE_CORRUPTED,
+        INVALID_ACTION
+    }
```

Comment:
Great use of enum for errors!
Using an enum here to represent error types really improves the code readability :))